skimage/util/tests/test_crop.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+
+from skimage import data
+from skimage.util.crop import crop
+
```

Comment:
There should also be some additional test functions added to test the logic for raising exceptions with invalid inputs. These can use the form:

```Python
with pytest.raises(ValueError):
    # code that should raise a ValueError here
```

Reply:
I have added few more testcases.